bugbug/model.py

```diff
@@ -17,7 +17,7 @@
 
 
 class Model:
-    def __init__(self, lemmatization=False):
+    def __init__(self, lemmatization=False, historical=False):
```

Comment:
We are not using the parameter here, so you can remove it.

Reply:
If I remove it, I get the following error:
` Model.__init__(self, lemmatization, historical)`

`TypeError: __init__() takes from 1 to 2 positional arguments but 3 were given`

since the constructor from this class is inherited by `bug.py`

Reply:
You'll need to remove the argument here, and stop passing it to Model.__init__ in bug.py